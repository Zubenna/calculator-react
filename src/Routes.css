.link-box {
  display: flex;
  justify-content: flex-end;
  position: relative;
  top: 1rem;
}

.link-menu {
  text-decoration: none;
  padding-right: 1rem;
  font-family: 'Courier New', Courier, monospace;
  font-size: 1.3rem;
}

.home-title,
.quote-title {
  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
  position: absolute;
  top: 0;
  left: 1rem;
  color: blue;
}

.home {
  margin-top: 3rem;
}

.home,
.quote p {
  font-size: 1.3rem;
  width: 80%;
  margin: 0 auto;
}

.home h3 {
  padding-top: 2rem;
}

.quote p {
  padding-top: 4rem;
}
