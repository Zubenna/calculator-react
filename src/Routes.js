import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import App from './components/App';
import Home from './components/Home';
import Quote from './components/Quote';

const Routes = () => (
  <BrowserRouter>
    <Switch>
      <Route path="/" component={Home} exact />
      <Route path="/app" component={App} exact />
      <Route path="/quote" component={Quote} exact />
    </Switch>
  </BrowserRouter>
);

export default Routes;
