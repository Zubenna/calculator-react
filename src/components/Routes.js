import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import App from './App';
import Home from './Home';
import Navbar from './Navbar';
import Quote from './Quote';
import '../Routes.css';

const Routes = () => (
  <BrowserRouter>
    <Navbar />
    <Switch>
      <Route path="/" component={Home} exact />
      <Route path="/app" component={App} exact />
      <Route path="/quote" component={Quote} exact />
    </Switch>
  </BrowserRouter>
);

export default Routes;
