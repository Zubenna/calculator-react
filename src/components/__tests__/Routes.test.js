import React from 'react';
import renderer from 'react-test-renderer';
import Routes from '../Routes';

describe('Routes', () => {
  it('Tests the rendering of components', () => {
    const tree = renderer
      .create(<Routes />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
