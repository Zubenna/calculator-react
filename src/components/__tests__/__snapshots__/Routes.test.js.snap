// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes Tests the rendering of components 1`] = `
Array [
  <div
    className="link-box"
  >
    <a
      className="link-menu"
      href="/"
      onClick={[Function]}
    >
      Home 
    </a>
    <a
      className="link-menu"
      href="/app"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="link-menu"
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </div>,
  <h1
    className="home-title"
  >
    Math Magicians
  </h1>,
  <div
    className="home"
  >
    <h3>
      Welcome to our page!
    </h3>
    <p>
      Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
    </p>
    <p>
      It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
    </p>
  </div>,
]
`;
